aten/src/ATen/native/native_functions.yaml

```diff
@@ -1816,6 +1816,7 @@
   dispatch:
     CPU: empty_strided_cpu
     CUDA: empty_strided_cuda
+    QuantizedCUDA: empty_strided_cuda_quantized
```

Comment:
nit: empty_strided_quantized_cuda

the naming convention is function_name_{backend_name}